overlord/ifacestate/ifacestate_test.go

```diff
@@ -2373,6 +2377,221 @@ func (s *interfaceManagerSuite) TestDoSetupSnapSecurityKeepsExistingConnectionSt
 	})
 }
 
+func (s *interfaceManagerSuite) TestReloadingConnectionsOnStartupUpdatesStaticAttributes(c *C) {
+	// Put a connection in the state. The connection binds the two snaps we are
+	// adding below. The connection contains a copy of the static attributes
+	// but refers to the "old" values, in contrast to what the snaps define.
+	s.state.Lock()
+	s.state.Set("conns", map[string]interface{}{
+		"consumer:plug producer:slot": map[string]interface{}{
+			"interface":   "test",
+			"plug-static": map[string]interface{}{"attr": "old-plug-attr"},
+			"slot-static": map[string]interface{}{"attr": "old-slot-attr"},
+		},
```

Comment:
Could you please add a "canary" connection here (for other snap)? I had one in the original test, the idea is to make sure it remains untouched after the connection we're iterested in gets updated.

Reply:
Sure, will do.